ega-charts/localega/values.yaml

```diff
@@ -39,6 +39,15 @@ config:
   tls_cert_ending: ".crt"
   tls_key_ending: ".key"
   tls_ca_root_file: "ca.crt"
+  # These are to be used when deploy for these services is set to false
+  external_filedatabase_host: ""
+  external_filedatabase_port: 9059
+  external_keys_host: ""
+  external_keys_port: 9095
+  external_dataedge_host: ""
+  external_dataedge_port: 9050
+  external_res_host: ""
+  external_res_port: 9090
```

Comment:
This doesn't make sense.
If these services are deployed in another cluster then they will have ingresses defined that means the ports will be 80 or 443 for all of them.
If they are set to use node ports then the port ranges is 30000-32000 ish.

Secondly the `external_` label is not relevant since it assumes that the current location is the base location.

Reply:
removed `external_` and defaulted port for all to `443` as `tls_certs` is enabled by default.